### DISCUSSION TITLE
dolby encoding engine

### DISCUSSION THREAD

**Discussion by CreaLogiciel** [quietvoid/dovi_tool#124] [Ideas]
hello quietvoid,
the dolby encoding engine software has been leaked. Inside there are functions for making Dolby Vision.
I don't know if you are aware of the availability of this software with Deew and if it can bring something to your great program.
Thanks again for your work

**Comment by quietvoid** [comment]
I don't want to deal with anything related to leaked professional tools.
From what I've been told, it doesn't offer much more than `dovi_tool`.
